docs/databases/index-content.md

```diff
@@ -2,15 +2,17 @@ This article provides an overview of the Azure database solutions described in A
 
 ![Diagram that contrasts relational database management system and big data solutions.](_images/architecture-design-types.svg)
 
-Azure Database solutions include both traditional relational database management system (RDBMS) and big data solutions.
+Azure Database solutions include traditional relational database management system (RDBMS) (this includes analytic solutions) and NoSQL solutions.
```

Comment:
```suggestion
Azure Database solutions include traditional relational database management system (RDBMS), data analytics solutions, and NoSQL solutions.
```

The reason you can't leave it like you have it, is because the image right above this contradicts what you said.  So unless there is an image change to go with this, we have to change how it's presented.  It's weird now that the image still doesn't match the text.  Any way you can get "big data" into the text as well?  That'll help!